fiscal_epos_print/static/src/js/pos_order_mgmt.js

```diff
@@ -0,0 +1,57 @@
+odoo.define("fiscal_epos_print.pos_order_mgmt", function (require) {
+    "use strict";
+
+    var core = require("web.core");
+    var pos_order_mgmt = require('pos_order_mgmt.widgets');
+    var epson_epos_print = require('fiscal_epos_print.epson_epos_print');
+    var _t = core._t;
+    var OrderListScreenWidget = pos_order_mgmt.OrderListScreenWidget;
+    var eposDriver = epson_epos_print.eposDriver;
+
+    OrderListScreenWidget.include({
+        _prepare_order_from_order_data: function (order_data, action) {
+            var order = this._super(order_data, action);
+            if (action === 'print') {
+                order.refund_report = order_data.refund_report;
+                order.refund_date = order_data.refund_date;
+                order.refund_doc_num = order_data.refund_doc_num;
+                order.refund_cash_fiscal_serial = order_data.refund_cash_fiscal_serial;
+            }
+            else if (action === 'return') {
+                order.refund_report = order_data.fiscal_z_rep_number;
+                order.refund_date = order_data.fiscal_receipt_date;
+                order.refund_doc_num = order_data.fiscal_receipt_number;
+                order.refund_cash_fiscal_serial = order_data.fiscal_printer_serial;
+            }
+            // for action === 'copy' we don't need to do anything
+
+            return order;
+        },
+
+        sendToFP90Printer: function(receipt, printer_options) {
+            for (var i = 0; i < printer_options.length; i++){
+                var printer_option = printer_options[i];
+                var fp90 = new eposDriver(printer_option, this);
+                fp90.printFiscalReceipt(receipt);
+            }
+        },
+
+        action_print: function (order_data, order) {
+            if (this.pos.config.printer_ip) {
+                if (order_data.fiscal_receipt_number) {
+                    this.pos.gui.show_popup('error', {
+                        'title': _t('Order already printed'),
+                        'body': order_data.pos_reference + _t(": order already has a fiscal number, ") + order_data.fiscal_receipt_number,
+                    });
+                    return;
+                }
```

Comment:
Mancherebbe
```
                this.chrome.loading_show();
                this.chrome.loading_message(_t('Connecting to the fiscal printer'));
```

Qui `order` non ha `export_for_printing`?

Reply:
è nella ``action_print()`` perchè così esporti l'ordine solo quando devi stampare.

Reply:
Che cosa è nella `action_print`?

Qui siamo nella `action_print` e mi pare manchi `this.chrome.loading_show`.
Quando mandi in stampa uno scontrino dalla lista di `pos_order_mgmt`, viene oscurato lo schermo in attesa della risposta della FP?

Reply:
Ho aggiunto la ``loading_show()`` nell'ultimo commit, mi pare ok. La ``export_for_printing()`` è subito dopo nella parte che manda tutto alla printer.